__tests__/cake.test.ts

```diff
@@ -1,14 +1,17 @@
-import * as path from 'path';
 import { exec } from '@actions/exec';
 import { which } from '@actions/io';
+import * as path from 'path';
+
 import * as cake from '../src/cake';
-import { ToolsDirectory } from '../src/toolsDirectory';
-import { CakeToolSettings } from '../src/cakeToolSettings';
 import { CakeArgument, CakeSwitch } from '../src/cakeParameter';
+import { CakeToolSettings } from '../src/cakeToolSettings';
+import { ToolsDirectory } from '../src/toolsDirectory';
 
 const pathToLocalToolsDirectory = path.join('path', 'to', 'tool');
 const pathToLocalTool = path.join(pathToLocalToolsDirectory, 'dotnet-cake');
 const dotnetManifestCake = 'dotnet tool run dotnet-cake';
+const pathToCsprojFile = path.join('build', 'Build.csproj');
```

Comment:
**nitpick:** it would be nice if this variable was declared right below `pathToLocalTool`.

Reply:
Done